src/components/Map/TopMenu/HamburgerMenu.tsx

```diff
@@ -198,13 +198,24 @@ export const HamburgerMenu = () => {
         open={opened}
         onClose={close}
       >
-        <ThemeSelection />
+        <AboutLink closeMenu={close} />
         <EditLink closeMenu={close} />
-        <StyledDivider />
         <InstallLink closeMenu={close} />
-        <AboutLink closeMenu={close} />
         <GithubLink closeMenu={close} />
         <StyledDivider />
+        {isOpenClimbing && [
+          <Typography variant="caption" ml={2} key="climbing-header">
+            Climbing
+          </Typography>,
+          <ClimbingAreasLink closeMenu={close} key="climbing-areas" />,
+          <ClimbingGradesTableLink closeMenu={close} key="climbing-grades" />,
+        ]}
```

Comment:
Ups, i accidentally removed this.

It was throwing a lot of errors in the console. Could you please take a look at it?